vadl/main/vadl/gcb/passes/relocation/model/ImplementedUserSpecifiedRelocation.java

```diff
@@ -125,4 +125,9 @@ public ElfRelocationName elfRelocationName() {
             + "_" + field.identifier.simpleName()
     );
   }
+
+  @Override
+  public String llvmKind() {
+    return kind.llvmKind();
+  }
```

Comment:
minor: just add a method to return the kind and all the users can call llvmKind themselves.